distribution/tools/server-cli/src/test/java/org/elasticsearch/server/cli/MachineDependentHeapTests.java

```diff
@@ -23,10 +23,12 @@
 // TODO: rework these tests to mock jvm option finder so they can run with security manager, no forking needed
 public class MachineDependentHeapTests extends ESTestCase {
 
+    private static final long SERVER_CLI_OVERHEAD = 120 * 1024L * 1024L;
```

Comment:
If the real one were package-private instead of protected, you could reference it from the test.

Reply:
👍 yes, let's avoid having to define this twice, and therefore have to change it in two places.